exceptions/messages.py

```diff
@@ -0,0 +1,70 @@
+"""Custom exceptions for the exceptions related to the messages file."""
+
+from collections.abc import Sequence
+
+__all__: Sequence[str] = (
+    "InvalidMessagesJSONFileError",
+    "MessagesJSONFileMissingKeyError",
+    "MessagesJSONFileValueError",
+)
+
+
+from classproperties import classproperty
+
+from exceptions import BaseTeXBotError, ImproperlyConfiguredError
```

Comment:
Use relative imports for importing base exception classes. I.e. `from .base import ...` rather than ~`from exceptions import ...`~ or ~`from exceptions.base import ...`~